packages/infra/lib/shared/rds.ts

```diff
@@ -5,6 +5,8 @@ import { Construct } from "constructs";
 import { RDSAlarmThresholds } from "../../config/aws/rds";
 import { mbToBytes } from "../shared/util";
 
+const DEFAULT_MIN_LOCAL_STORAGE_MB_ALARM = 10_000;
```

Comment:
this reminds me that we should probably raise an issue to not store WH payloads in the DB, but rather pointers to S3 files, since we know how large these can get

Reply:
Good call: https://github.com/metriport/metriport-internal/issues/1748